cmake/FindPackageHandleStandardArgs.cmake

```diff
@@ -0,0 +1,605 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
@shgandhi, this file is a part of a standard cmake installation, there's no need to put it here, also it may change when upgrading cmake version